spec/tbd_tests_spec.rb

```diff
@@ -2392,4 +2392,1614 @@
       expect(surface[:heatloss]).to be_within(TOL).of(-0.20) if id == ids[:l]
     end
   end
+
+  it "can process thermal bridging and derating: LoScrigno" do
```

Comment:
This test is part of the TBD Test Suite. Holding it here temporarily until the ASHRAE 90.1-related adaptations are completed.